static/site.css

```diff
@@ -834,14 +838,29 @@ ul.guide li {
 }
 
 #settings {
+  overflow-y: auto;
+  max-height: 500px;
```

Comment:
Maybe don't define stuffs in absolute number of pixels? Responsive design and all 😅